pangea-sdk/v3/service/authn/api.go

```diff
@@ -0,0 +1,1419 @@
+	Scopes        *Scopes      `json:"scopes,omitempty"`
+}
+
+type UserCreateResult struct {
+	ID           string        `json:"id"`
+	Email        string        `json:"email"`
+	Profile      ProfileData   `json:"profile"`
+	IDProviders  []string      `json:"id_providers"`
+	RequireMFA   bool          `json:"require_mfa"`
+	Verified     bool          `json:"verified"`
+	LastLoginAt  *string       `json:"last_login_at,omitempty"`
+	Disabled     bool          `json:"disabled"`
+	MFAProviders []MFAProvider `json:"mfa_providers,omitempty"`
+	CreatedAt    string        `json:"created_at"`
+}
+
+// @summary Create User
+//
+// @description Create a user.
+//
+// @operationId authn_post_v1_user_create
+//
+// @example
+//
+//	profile := &authn.ProfileData{
+//		"first_name": "Joe",
+//		"last_name": "User",
+//	}
+//
+//	input := authn.UserCreateRequest{
+//		Email: "joe.user@email.com",
+//		Authenticator: "My1s+Password",
+//		IDProvider: authn.IDPPassword,
+//		Profile: profile,
+//	}
+//
+//	resp, err := authncli.User.Create(ctx, input)
+func (a *User) Create(ctx context.Context, input UserCreateRequest) (*pangea.PangeaResponse[UserCreateResult], error) {
+	return request.DoPost(ctx, a.Client, "v2/user/create", &input, &UserCreateResult{})
+}
+
+type UserDeleteRequest struct {
+	// Base request has ConfigID for multi-config projects
+	pangea.BaseRequest
+
+	Email string `json:"email,omitempty"`
+	ID    string `json:"id,omitempty"`
+}
+
+type UserDeleteResult struct {
+}
+
+// @summary Delete User
+//
+// @description Delete a user by email address.
+//
+// @operationId authn_post_v1_user_delete
+//
+// @example
+//
+//	input := UserDeleteRequest{
+//		Email: "joe.user@email.com",
+//	}
+//
+//	authncli.User.Delete(ctx, input)
+func (a *User) Delete(ctx context.Context, input UserDeleteRequest) (*pangea.PangeaResponse[UserDeleteResult], error) {
+	return request.DoPost(ctx, a.Client, "v2/user/delete", &input, &UserDeleteResult{})
+}
+
+type UserItem struct {
+	ID                      string          `json:"id"`
+	Email                   string          `json:"email"`
+	Profile                 ProfileData     `json:"profile"`
+	Verified                bool            `json:"verified"`
+	Disabled                bool            `json:"disabled"`
+	AcceptedEulaID          *string         `json:"accepted_eula_id,omitempty"`
+	AcceptedPrivacyPolicyID *string         `json:"accepted_privacy_policy_id,omitempty"`
+	LastLoginAt             *string         `json:"last_login_at,omitempty"`
+	CreatedAt               string          `json:"created_at"`
+	LoginCount              int             `json:"login_count"`
+	LastLoginIP             *string         `json:"last_login_ip,omitempty"`
+	LastLoginCity           *string         `json:"last_login_city,omitempty"`
+	LastLoginCountry        *string         `json:"last_login_country,omitempty"`
+	Authenticators          []Authenticator `json:"authenticators"`
+}
+
+type UserUpdateRequest struct {
```

Comment:
```go
type UserUpdateRequest struct {
	// Base request has ConfigID for multi-config projects
	pangea.BaseRequest

	ID       *string `json:"id,omitempty"`
	Email    *string `json:"email,omitempty"`
	Disabled *bool   `json:"disabled,omitempty"`
}
```

Needs to be updated to:
![Screenshot 2023-10-19 at 4 04 26 PM](https://github.com/pangeacyber/pangea-go/assets/10459856/2ec6e4b0-b192-4356-b0fb-9c91a5c9c372)
![Screenshot 2023-10-19 at 4 04 32 PM](https://github.com/pangeacyber/pangea-go/assets/10459856/a2c8d10c-5454-4661-a147-d2953f13efcb)

Reply:
Disregard, I was looking at the wrong endpoint. This is indeed correct.